README.md

```diff
@@ -150,3 +150,33 @@ The following parameters affect high availability:
 
   - `vip.vip` - Which IP to use as a VIP that is traded between the
     two nodes.
+
+### HA Failure Modes
+
+Our HA solution is focused on preventing downtime in the face of
+upgrades or other single-node failure. As such, we do not attempt to
+solve scenarios where the two databases cannot communicate with one
+another (e.g. network failure). In this case, it is possible that the
```

Comment:
In HA parlance, this is called a _partition_; we should use the standard terminology.